### PR TITLE
Switch to CCZero

### PR DESCRIPTION
**Pull request by egonw**
Implements https://github.com/semanticchemistry/semanticchemistry/issues/47

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Creative Commons CC0 1.0 Universal
```
Comment (cthoyt) [inline LICENSE.md:1]: better to do plain text license that github can pick up because it has standard text

File: ontology/cdk.owl [hunk 2]
```diff
@@ -42,7 +42,7 @@
         <dc:format rdf:datatype="&xsd;string">application/rdf+xml</dc:format>
         <dc:title rdf:datatype="&xsd;string">CDK descriptor algorithm implementations</dc:title>
         <dc:language rdf:datatype="&xsd;string">en</dc:language>
-        <dc:rights>http://creativecommons.org/licenses/by/3.0</dc:rights>
+        <dc:rights>https://creativecommons.org/publicdomain/zero/1.0/legalcode</dc:rights>
```
Comment (cthoyt) [inline ontology/cdk.owl:45]:
```suggestion
         <dc:rights>https://creativecommons.org/publicdomain/zero/1.0</dc:rights>
```

File: ontology/cheminf-external.owl [hunk 3]
```diff
@@ -40,7 +40,7 @@
         <dc:format rdf:datatype="&xsd;string">application/rdf+xml</dc:format>
         <dc:language rdf:datatype="&xsd;string">en</dc:language>
         <protege:defaultLanguage rdf:datatype="&xsd;string">en</protege:defaultLanguage>
-        <dc:rights>http://creativecommons.org/licenses/by/3.0</dc:rights>
+        <dc:rights>https://creativecommons.org/publicdomain/zero/1.0/legalcode</dc:rights>
```
Comment (cthoyt) [inline ontology/cheminf-external.owl:43]:
```suggestion
        <dc:rights>https://creativecommons.org/publicdomain/zero/1.0</dc:rights>
```

File: ontology/cdk.owl (hunk 2 above)
Comment (cthoyt) [reply ontology/cdk.owl:45]: Note that all of these should probably be `dc:license` based on #31

File: LICENSE.md (hunk 1 above)
Comment (egonw) [reply LICENSE.md:1]: I think it will do fine. I'll make sure to check when merged.

File: ontology/cdk.owl (hunk 2 above)
Comment (egonw) [reply ontology/cdk.owl:45]: Fixed.

File: ontology/cheminf-core.owl [hunk 4]
```diff
@@ -44,7 +44,7 @@
         <dc:title rdf:datatype="&xsd;string">chemical information ontology (cheminf) - information entities about chemical entities</dc:title>
         <dc:language rdf:datatype="&xsd;string">en</dc:language>
         <dc:contributor>Leonid Chepelev</dc:contributor>
-        <dc:rights>http://creativecommons.org/licenses/by/3.0</dc:rights>
+        <dc:rights>https://creativecommons.org/publicdomain/zero/1.0/legalcode</dc:rights>
```
[the remark this answers is not in the document]
Comment (egonw) [reply ontology/cheminf-core.owl:47]: Yeah, thanks for that. There is a controlled list of license IRIs, and I should have checked that. I guess it was too hot. Thanks for being critical!

File: ontology/cheminf.owl [hunk 5]
```diff
@@ -56,7 +56,7 @@
 specified URI = http://semanticscience.org/resource/
 Followed by &apos;/&apos;, End with : AutoID
 AutoID - digit count = 6, prefix = CHEMINF_</rdfs:comment>
-        <dc:rights>http://creativecommons.org/licenses/by/3.0</dc:rights>
+        <dc:rights>https://creativecommons.org/publicdomain/zero/1.0/legalcode</dc:rights>
```
[the remark this answers is not in the document]
Comment (egonw) [reply ontology/cheminf.owl:59]: PS. the IRI is with http instead of https

**Review by cthoyt — APPROVED** [review]
not sure about http/https but good enough for now

**Comment by StroemPhi** [comment]
creativecommons.org redirects to https automatically so no need to worry, right?

**Pull request merged**